addons/digest/data/digest_data.xml

```diff
@@ -11,7 +11,7 @@
         <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=no;"/>
         <meta http-equiv="X-UA-Compatible" content="IE=9; IE=8; IE=7; IE=EDGE" />
         <style type="text/css">
-            <t t-set="color_company" t-value="company.secondary_color or '#875a7b'"/>
+            <t t-set="color_company" t-value="company.email_secondary_color or '#875a7b'"/>
```

Comment:
Upgrade question: Aren't we supposed to force the update of those templates? (maybe with if_unchanged as it's not a critical change).